### ISSUE TITLE
Partners Page - Change text content

### ISSUE THREAD

**Issue by Code4SocialGood** [Code4SocialGood/c4sg-web#1661]
[C4SG Partners Page.docx](https://github.com/Code4SocialGood/c4sg-web/files/1490164/C4SG.Partners.Page.docx)

**Comment by SouravInsights** [comment]
@Code4SocialGood  could you please elaborate the tasks and issues so that I can work on it.

**Comment by Code4SocialGood** [comment]
@SouravInsights Please find below the details of change request:

There are 6 sections on the Partners page. 
Each section contains a picture, a section title, and a section description.
**Please change the section title and section description, with the text provided below.**
Please note that in the current page, it doesn't show section description. The code to show section description should already be there but have been commented out. Please uncomment out  so that description will show.

```
Tech Company
Industry companies are encouraged to get involved as mentors and advisors. Options include sponsoring certain projects and providing guidance to our clients and volunteers. Monetary donations are accepted as well.

School Clubs 
Is your student club seeking more volunteer hours? Partner with us and we will find projects right for your team. At C4SG, schedules are adjustable, meaning you can work within the time frame you need.

User Groups 
With our wide variety of projects listed, we can find a venture perfect for the focus of your user group. Whatever your specialty may be, there is a project right for you.

Tech Nonprofits 
If you are a tech nonprofit like us, we are always open to joint ventures. Together, we can support large-scale ventures or high-tech projects. Community events are always an option too!

Media 
As a media outlet, you can help us by promoting our initiative. Plugs for the website are always appreciated, in addition to advertisements and features about our cause. Help us spread the good of tech volunteering!

Other Groups  
Unable to find a listing right for your group? Message us and we can work something out!
 
```

**Comment by NjeriNjoroge** [comment]
Hi, @Code4SocialGood I have already cloned the project and I would like to close this issue. However, the test for Frontend isn't working. Are there instructions for users on Ubuntu?

**Comment by Code4SocialGood** [comment]
Resolved in PR #1739
